src/fn/upload-app-settings.js

```diff
@@ -1,14 +1,21 @@
 const api = require('../lib/api');
-const environment = require('../lib/environment');
 const fs = require('../lib/sync-fs');
+const path = require('path');
+const { info, error } = require('../lib/log');
+const environment = require('../lib/environment');
 
-module.exports = () => {
-  const settings = fs.read(`${environment.pathToProject}/app_settings.json`);
+module.exports = async () => {
```

Comment:
This looks like you're adding the async flag but you're not using await. The async is therefore unnecessary (or you can use await instead of `.then()`

Reply:
Fixed